mmv1/third_party/terraform/tests/resource_cloudids_endpoint_test.go

```diff
@@ -0,0 +1,95 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestCloudIdsEndpoint_basic(t *testing.T) {
```

Comment:
Following the question above about the example, if we can remove the `skip_test` there, we can repurpose this handwritten test to be an update test where we try changing every updatable field.

Reply:
Got it. Also see comments at skip_test